cdf_fabric_replicator/extractor.py

```diff
@@ -50,8 +93,8 @@ def get_service_client_token_credential(self, account_name: str) -> DataLakeServ
 
         return service_client
 
-    def download_files_from_abfss(self, abfss_directory: str) -> None:
-        container_name, account_name, file_path = self.parse_abfss_url(abfss_directory)
+    def download_files_from_abfss(self, file_path: str) -> None:
```

Comment:
Your method called download_files_from_abfss calls code to upload files to CDF. Uploading should be handled in a different method or this method should be more appropriately named

Reply:
Done